.github/workflows/zjit-macos.yml

```diff
@@ -38,9 +38,8 @@ jobs:
           - test_task: 'ruby' # build test for combo build
             configure: '--enable-yjit --enable-zjit'
 
-          - test_task: 'test-all'
```

Comment:
I'm not sure if it's worth testing `test_zjit` by itself on CI when we'll be running it in `zjit-test-all` anyway. Also the `test-all` name will be confusing along with `zjit-test-all`.